test/com/cloudogu/ces/cesbuildlib/K3dTest.groovy

```diff
@@ -9,7 +11,7 @@ import static org.mockito.ArgumentMatchers.anyString
 import static org.mockito.Mockito.mock
 import static org.mockito.Mockito.when
 
-class K3dTest extends GroovyTestCase {
+class K3dTest {
```

Comment:
Finally arriving in year 2010 with tests that no longer extend other testcases :laughing: